pages/components/box.html

```diff
@@ -0,0 +1,164 @@
+<html lang="en">
+  <%- include('head.html'); -%>
+  <body>
+    <f-docs-template>
+      <%- include('nav.html'); -%>
+      <main slot="content">
+        <h1 class="mb-16">Box</h1>
+        <p>Box is a layout component used for separating content areas on a page.</p>
+
+        <h2 class="mt-24 mb-16">Props</h2>
+        <table class="w-full docs-table">
+          <thead>
+            <tr class="text-left text-gray-400 uppercase">
+              <th>prop</th>
+              <th>type</th>
+              <th>default</th>
+            </tr>
+          </thead>
+          <tbody class="align-top">
+            <tr>
+              <td>bleed</td>
+              <td>
+                <div>boolean</div>
+                <div class="annotation">Will make a box full-width on mobile</div>
+              </td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>bordered</td>
+              <td>boolean</td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>info</td>
+              <td>
+                <div>boolean</div>
+                <div class="annotation">Will style the box with light blue colors</div>
+              </td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>neutral</td>
+              <td>
+                <div>boolean</div>
+                <div class="annotation">Will style the box with light gray colors</div>
+              </td>
+              <td>false</td>
+            </tr>
+          </tbody>
+        </table>
```

Comment:
This is super nice. I should go back and improve the props on the card, toast and broadcast component doc pages to look like this